staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -707,10 +705,9 @@ type ReplicaSetSpec struct {
 	MinReadySeconds int32 `json:"minReadySeconds,omitempty" protobuf:"varint,4,opt,name=minReadySeconds"`
 
 	// Selector is a label query over pods that should match the replica count.
-	// If the selector is empty, it is defaulted to the labels present on the pod template.
 	// Label keys and values that must match in order to be controlled by this replica set.
+	// Must be specified that matches the labels present on the pod template.
```

Comment:
`It must match the pod template's labels.` is clear enough.

Reply:
@crimsonfaith91 Addressed